### ISSUE TITLE
Payment Requests

### ISSUE THREAD

**Issue by oojacoboo** [appditto/natrium_wallet_flutter#27]
I’d be amazing if Natrium could generate a deeplink payment request in the form of a URL that could be “shared” via the native iOS sharing tool.  So you could message however needed and when clicked would instantly open Natrium, or whatever wallet said person is using that supports Nano, with the amount of the payment pre-populated.  

It’d be even better if it’s possible to pass a message along with the request (eg. “For tacos last night”).

**Comment by bbedward** [comment]
Agreed, would be a great feature to integrate with the nanotipbot deep links as well. Maybe @mitche50 could include a memo field on the page for when Natrium isn't installed.

**Comment by mitche50** [comment]
Easy enough to add - I'll edit the gist and add it to my website tonight.
